index.ts

```diff
@@ -168,6 +172,16 @@ for (const [key, data] of yamlEntries('features')) {
         }
     }
 
+    // Ensure that version ranges are only used for prehistoric features.
+    if (data.status?.support) {
+        const versions: string[] = Object.values(data.status.support);
+        if (versions.some((v) => v.startsWith('≤'))) {
+            if (data.status.baseline_low_date !== earliestBaselineLowDate) {
```

Comment:
Should I change this to allow for ranges when the Baseline low date is before 2016? I'm on the fence.

Reply:
I like this being a more specific thing to start, then expanded to later uses later (as needed).